### DISCUSSION TITLE
Tom Explorer Remove Filters

### DISCUSSION THREAD

**Discussion by databayes** [TabularEditor/TabularEditor3#1190] [Ideas]
### Is your feature request related to a problem? Please describe

I forget that I have filtered Object Type in the TOM explorer and then get frustrated when I can't find something.  Also takes 3 clicks to then click on Filter, Select All and then Click Ok.

### Describe the solution you'd like

Can you add a remove filters icon to remove the applied filters?

### Goals

1.
2.

**Comment by otykier** [comment]
Maybe I'm misunderstanding this ask, but we have a "Clear Filter" option in the right-click menu. Is this sufficient?

<img width="455" alt="image" src="https://github.com/TabularEditor/TabularEditor3/assets/8976200/dbe1dc5c-8e37-4b80-91b6-97102cb98776">

**Reply by databayes** [reply]
Perfect, thank you
